engine/src/main/java/com/sandboni/core/engine/finder/bcel/visitors/AnnotationUtils.java

```diff
@@ -24,9 +27,8 @@ public static String getTypeSignature(ConstantPool constantPool, int typeIndex)
     }
 
     public static String getAnnotationParameter(ConstantPool constantPool, Supplier<AnnotationEntry[]> annotationSupplier, String annotationName, String... parameterNames) {
-        AnnotationEntry annotation = getAnnotation(constantPool, annotationSupplier, annotationName);
-
-        return annotation == null ? "" : getAnnotationParameter(annotation, parameterNames);
+        Optional<AnnotationEntry> annotation = getAnnotation(constantPool, annotationSupplier, annotationName);
+        return annotation.isPresent() ? getAnnotationParameter(annotation.get(), parameterNames) : "";
```

Comment:
the better way: 
annotation.map(annotationValue -> getAnnotationParameter(annotationValue, paramNames)).orElse("");

Reply:
Gotcha, updated the PR. Thanks !